### ISSUE TITLE
Easy locate python path

### ISSUE THREAD

**Issue by Pullusb** [tin2tin/Python_Stylechecker_for_Blender#1]
Hi, just to mention something you can add in your readme to help users locate easily their python bin :
"
Running `bpy.app.binary_path_python` in blender interactive console will display the path to blender python binary.
"

And Just food for thought : Since you can get the python path and have similar pip commands for all OS, I wonder if you can auto-install the module at first register (with a popup asking if he agree).

**Comment by tin2tin** [comment]
Thanks. I added it to the readme. 

It should properly be converted into a multi-file add-on, and the module could be added to the add-on download. But I really prefer to code in the Text Editor, so I do not have to wait each time Blender needs to be restarted when using an external editor and dealing with multi-file add-ons form inside Blender is tedious. But having it as a multi-file add-on would be more user friendly for sure. 

Apparently there is also a way here to make the add-on install modules: https://developer.blender.org/T71420#807828

But it would be much better if module installation was added directly to the Blender API.
